src/ape/managers/project.py

```diff
@@ -470,10 +470,9 @@ def __getattr__(self, attr_name: str) -> ContractContainer:
         """
 
         contracts = self.load_contracts()
+
         if attr_name in contracts:
             contract_type = contracts[attr_name]
-        elif attr_name in self.dependencies:
-            contract_type = self.dependencies[attr_name]  # type: ignore
```

Comment:
Wait, is this change removing the support for `project.MyDependency`?

Reply:
The contract types are available on the project already, so this `if` statement would never get called if given the contract name of a dependency.

However - before you could do `project.OpenZeppelin` and have access to the project manifest. Do we still want to support that? We could still do the `from_manifest()` idea too, I just didn't know if it was as important knowing we can access any contact type we need.

Yes, I am confused to. I would like to know the ideal situation so we can strive for that.

Reply:
Ah yes, so I think the original design here was thinking that `dependencies: Dict[str, ProjectManager]`, so that if you did `project.my_dependency.DependencyContractType` it would first do `project.__getattr__` to look up the dependency and return that `ProjectManager` instance, and then next do `my_dependency.__getattr__` to get the contract inside that `ProjectManager` instance.

I think we changed things around a lot here where `dependencies: Dict[str, PackageManifest]` instead, causing the issue since `PackageManifest.__getattr__` returns an `ethpm_types.ContractType` data model instead